Olympus/modfile_olympus.txt

```diff
@@ -0,0 +1,11 @@
+
+:: Import "CerberusAssist/CerberusAssist.lua"
```

Comment:
Cerberus and Extender are not loaded so we are good. They'll be uncommented on the next Release.